modules/cnf-deploying-the-du-infrastructure-profile.adoc

```diff
@@ -0,0 +1,269 @@
+      check_fup_sync 0
+      #
+      # Servo Options
+      #
+      pi_proportional_const 0.0
+      pi_integral_const 0.0
+      pi_proportional_scale 0.0
+      pi_proportional_exponent -0.3
+      pi_proportional_norm_max 0.7
+      pi_integral_scale 0.0
+      pi_integral_exponent 0.4
+      pi_integral_norm_max 0.3
+      step_threshold 0.0
+      first_step_threshold 0.00002
+      max_frequency 900000000
+      clock_servo pi
+      sanity_freq_limit 200000000
+      ntpshm_segment 0
+      #
+      # Transport options
+      #
+      transportSpecific 0x0
+      ptp_dst_mac 01:1B:19:00:00:00
+      p2p_dst_mac 01:80:C2:00:00:0E
+      udp_ttl 1
+      udp6_scope 0x0E
+      uds_address /var/run/ptp4l
+      #
+      # Default interface options
+      #
+      clock_type OC
+      network_transport UDPv4
+      delay_mechanism E2E
+      time_stamping hardware
+      tsproc_mode filter
+      delay_filter moving_median
+      delay_filter_length 10
+      egressLatency 0
+      ingressLatency 0
+      boundary_clock_jbod 0
+      #
+      # Clock description
+      #
+      productDescription ;;
+      revisionData ;;
+      manufacturerIdentity 00:00:00
+      userDescription ;
+      timeSource 0xA0
+  recommend:
+  - profile: "slave"
+    priority: 4
+    match:
+    - nodeLabel: "ptp/slave"
+----
+<1> The interface selected needs to match the Linux interface name.
+<2> `-2` configures Ethernet encapsulation of PTP.  `--summary_interval -4` sets the logging interval.
+This is currently set to match `logSyncInterval -4`.
+<3> `-n 24` must match the `domainNumber 24`.
+<4> `domainNumber 24` must match the `-n 24`.
+<5> These variables are set to enable the G.8275.1 profile for PTP.
+
+[id="cnf-creating-the-sriov-operator-and-associated-profiles_{context}"]
+== Creating the SR-IOV operator and associated profiles
+
+. Create  the `kustomization.yaml` for the `sriov-network-node-policy-dpdk-mh.yaml` and `sriov-network-mh.yaml` files, for example:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - sriov-network-node-policy-dpdk-mh.yaml
+  - sriov-network-mh.yaml
+----
+
+. Create the `sriov-network-node-policy-dpdk-mh.yaml` file, for example:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+# This works for Intel based NICs. <1>
+# For Mellanox please change to:
+#     deviceType: netdevice
+#     isRdma: true
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+# The exact physical function name must match the hardware used
+    pfNames: ["ens1f1"] <2>
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 4
+  priority: 10
+  resourceName: mh_u_site_1_fqdn_worker1
+----
+<1> This file works for Intel and must change for Mellanox.
```

Comment:
```suggestion
<1> This file works for Intel and must change for Mellanox, as described in SR-IOV configuration notes chapter.
```
(please link to the chapter in  modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc)

Reply:
We can't use internal xrefs (our current build machine won't render them). I'll have to use hard text.